frontend/src/components/mine/Permit/MinePermitTable.js

```diff
@@ -87,17 +88,15 @@ const columns = [
             <button
               type="button"
               className="full add-permit-dropdown-button"
-              onClick={(event) =>
-                record.openAddAmalgamatedPermitModal(event, text.guid, text.permit_no)
-              }
+              onClick={(event) => record.openAddAmalgamatedPermitModal(event, record.permit)}
             >
               <div>
                 <Icon
                   type="plus"
                   className="padding-small add-permit-dropdown-button-icon"
                   theme="outlined"
                 />
-                {text.hasAmalgamated ? "Add permit amendment" : "Add amalgamated permit"}
+                {text.hasAmalgamated ? "Add permit amendment" : "Amalgamate permit"}
```

Comment:
The code is correct. If the permit has an amalgamated amendment, new amendments are amalgamated by default and the text "Add permit amendment" is desired. Otherwise, "Amalgamate permit" is the action to add a new amalgamated amendment.